cmd/template/cluster/flag.go

```diff
@@ -129,10 +135,12 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.OpenStack.NodeCIDR, flagOpenStackNodeCIDR, "", "CIDR used for the nodes.")
 
 	// OpenStack App only.
+	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppCatalog, flagClusterAppCatalog, "giantswarm-catalog", "Cluster App version to be installed. (OpenStack App CR only).")
```

Comment:
Changed default to `giantswarm` catalog in https://github.com/giantswarm/cluster-openstack/pull/24